apps/aecore/src/aec_conductor.erl

```diff
@@ -628,13 +658,14 @@ post_block_worker(From, Block) ->
             end
     end.
 
+handle_synced_block_reply({block_added, _Block, From}, State) ->
+    common_block_added_reply(From, State);
+handle_synced_block_reply(Result, State) ->
+    handle_post_block_reply(Result, State).
+
 handle_post_block_reply({block_added, Block, From}, State) ->
     aec_events:publish(block_received, Block),
```

Comment:
I see the difference between post_block and synced_block is this event publishing. Ok.